docs/doxygen/overviews/archive.h

```diff
@@ -65,12 +67,12 @@ ownership).
 Reading from the input stream then returns the entry's data. Eof() becomes
 @true after an attempt has been made to read past the end of the entry's data.
 
-When there are no more entries, GetNextEntry() returns @NULL and sets Eof().
+When there are no more entries, GetNextEntry() returns @nullptr and sets Eof().
```

Comment:
I've changed `@NULL` alias in `docs/doxygen/Doxyfile` to expand to `nullptr` too to avoid having to change all of its existing occurrences. I don't know if it's too confusing to keep using it, but even if not, we should replace all of its occurrences with `@nullptr` at once, otherwise we're going to end up with an unholy mix of the two which would be even worse.

IOW I'd like to
```suggestion
When there are no more entries, GetNextEntry() returns @NULL and sets Eof().
```
at least for now.